definition/definition.go

```diff
@@ -23,7 +23,15 @@ import (
 	"github.com/caicloud/nirvana/router"
 )
 
-type Operator func(ctx context.Context, object interface{}) (interface{}, error)
+type Operator interface {
+	Operate(ctx context.Context, object interface{}) (interface{}, error)
```

Comment:
A `Kind()` method is needed. It indicates the kind of operator. Other tools can recognize an operator by its kind.